packages/cli-lib/api/fileMapper.js

```diff
@@ -181,7 +181,12 @@ async function trackUsage(eventName, eventClass, meta = {}, accountId) {
     eventClass,
     meta,
   };
-  const path = `${FILE_MAPPER_API_PATH}/cms-cli-usage`;
+  const endpointMapping = {
+    'cli-interaction': 'cms-cli-usage',
+    'vscode-extension-interaction': 'vscode-extension-usage',
```

Comment:
Maybe create an enum for the possible `eventName` values?

Reply:
Thanks @gcorne. I updated this if you don't mind taking another look I'd appreciate it